modules/react/modal/lib/ModalOverlay.tsx

```diff
@@ -44,22 +44,24 @@ const Container = styled(Box)<StyledType>({
 // positioning, but seems to when using flexbox centering. This messes up Popper calculations inside
 // the Modal. The centering container forces a "center" pixel calculation by making sure the width
 // is always an even number
-const CenteringContainer = styled('div')({
+const ResponsiveCenteringContainer = styled('div')(({theme}) => ({
   maxHeight: '100vh',
   display: 'flex',
   position: 'absolute',
   left: 0,
   top: 0,
-  alignItems: 'center',
   justifyContent: 'center',
-
+  alignItems: 'center',
   // IE11 fix for setting min-height in a flex container
   ':before': {
     display: 'block',
     content: "''",
     height: '100vh',
   },
-});
+  [theme.canvas.breakpoints.down('s')]: {
+    alignItems: 'end',
+  },
```

Comment:
On smaller screen sizes, we align the modal to the bottom instead of the center.